k8s.gcr.io/README.md

```diff
@@ -10,13 +10,31 @@ repositories used to publish official container images for Kubernetes.
 
 ## Staging repos
 
-Each "project" (as decided by people) that feeds into Kubernetes' main
-image-serving system (k8s.gcr.io) gets a staging repository.  Each staging
-repository is governed by a googlegroup, which grants push access to that
-repository.
-
-Project owners can push to their staging repository and use the image promoter
-to promote images to the main serving repository.
+Kubernetes subprojects may use a dedicated staging GCP project to build and
+host container images. We refer to the GCR provided by each staging project
+as a staging repository. Images are promoted from staging repositories into
+the main Kubernetes image-serving system (k8s.gcr.io).
+
+Access to each staging project is governed by a googlegroup, which grants the
+ability to manually trigger GCB or push container images in the event that
+automated builds via something like prow.k8s.io are not setup or are broken.
+
+### Requirements
+
+The rule of thumb is that staging repositories should be used to host
+artifacts produced by code that is part of the Kubernetes project. In other
```

Comment:
```suggestion
artifacts produced by code that is part of the Kubernetes community. In other
```

Forgive my poor english but I feel like _Kubernetes project_ don't cover subprojects.

Reply:
I have a problem with it the other way, where community could imply the broader ecosystem of projects and users that depend upon Kubernetes.  I used "project" here in the same context as "Kubernetes as a project is owned by the CNCF".  That covers all of the subprojects that are part of kubernetes.

I could capitalize Project to make it clearer I'm trying to cover a very specific definition here?

Reply:
+1 for _Kubernetes Project_.

Reply:
I updated to define as present in one of the project-owned github organizations